cohesity_management_sdk/models/a_w_s_fleet_params.py

```diff
@@ -0,0 +1,80 @@
+# -*- coding: utf-8 -*-
+# Copyright 2020 Cohesity Inc.
+
+import cohesity_management_sdk.models.fleet_tag
```

Comment:
y not just name aws_fleet_params.py ?

Reply:
there is already a module names as aws_fleet_params.py
We have two definitions AWSFleetParams and AwsFleetParams

Reply:
aren't they the same models ?